interface/wx/hwinfo.h

```diff
@@ -0,0 +1,140 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        hwinfo.h
+// Purpose:     interface of wxSystemHardwareInfo
+// Author:      wxWidgets team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+
+/**
+    @class wxSystemHardwareInfo
+
+    This class holds information about the client's system hardware
+    (e.g., physical memory, battery, CPU, etc.).
+
+    All member functions are static and can accessed as such:
+    @code
+        wxMemorySize amountOfRAM = wxSystemHardwareInfo::GetPhysicalMemory();
+        wxLogMessage("Total RAM installed: %s.",
+                     wxFileName::GetHumanReadableSize(amountOfRAM.GetValue()));
+    @endcode
+
+    @since 3.3.0
+
+    @library{wxbase}
+    @category{cfg}
+
+    @see wxPlatformInfo(), @ref group_funcmacro_networkuseros
+*/
+class wxSystemHardwareInfo
+{
+public:
+
+    /**
+        @name CPU information
+    */
+    //@{
+
+    /**
+        Returns the number of CPU cores.
+    */
+    static int GetCPUCount();
+
+    /**
+        Returns the CPU architecture name.
+
+        This can be, for example, "x86_64", "arm64", or "i86pc".
+        The name for the same CPU running on the same hardware
+        can vary across operating systems.
+
+        The returned string may be empty if the CPU architecture
+        couldn't be recognized.
+
+        @see wxGetCpuArchitectureName()
```

Comment:
```suggestion
        @see ::wxGetCpuArchitectureName()
```
Just a nitpick, to make it same as in the other methods.